ubuntudesign/vanilla-framework
# Make all SVGs inline

## Done

Convert all asset server based SVG images inline SVGs

Create new tools-nav-icon-color function for nav icons

## QA

To QA the nav menu link (aka burger) change $brand-color to a dark colour e.g #333, run gulp build, the icon should now be white, change it back to a lighter colour and it should change to #333 

Check each of the replaced SVGs and make sure they are loading
